### ISSUE TITLE
What is _AMT_thd_MarriedS?

### ISSUE THREAD

**Issue by MattHJensen** [open-source-economics/Tax-Calculator#1108]
Is the _AMT_thd_MarriedS up to date?

I don't understand the parameter well enough to tell whether it is up to date by examining the forms. 

    "_AMT_thd_MarriedS": {
        "long_name": "Extra alminc for married sep",
        "description": "",
        "irs_ref": "Form 6251, line 28, instruction.",
        "notes": "",
        "start_year": 2013,
        "col_var": "",
        "row_var": "FLPDYR",
        "row_label": ["2013",
                      "2014"],
        "cpi_inflated": true,
        "col_label": "",
        "value": [40400,
                  41050]
    },

```
c62100 += cmbtp  # add income not in AGI but considered income for AMT
    if MARS == 3 or MARS == 6:
        amtsepadd = max(0.,
                        min(AMT_thd_MarriedS, 0.25 * (c62100 - AMT_em_pe)))
    else:
        amtsepadd = 0.
    c62100 = c62100 + amtsepadd  # AMT taxable income, which is line28
```

Assigning this to myself but welcome input if you know the answer off of the top of your head.

**Comment by MattHJensen** [comment]
Similarly, why do we need a separate AMT exemption phaseout end for married filing separately (_AMT_em_pe). Why don't the AMT exemption phaseout start (_AMT_em_ps), the AMT exemption phaseout rate (_AMT_prt), and AMT exemption (_AMT_em) jointly determine this? 



    "_AMT_em_pe": {
        "long_name": "AMT exemption phaseout ending AMT taxable income (Married filling Separately)",
        "description": "The AMT exemption is entirely disallowed beyond this AMT taxable income level for individuals who are married but filing separately.",
        "irs_ref": "Form 6251, line 28, instruction.",
        "notes": "",
        "start_year": 2013,
        "col_var": "",
        "row_var": "FLPDYR",
        "row_label": ["2013",
                      "2014"],
        "cpi_inflated": true,
        "col_label": "",
        "value": [238550,
                  242450]
    },


    "_AMT_em_ps": {
        "long_name": "AMT exemption phaseout start",
        "description": "AMT exemption starts to decrease when AMT taxable income goes beyond this threshold.",
        "irs_ref": "Form 1040, line 45, instruction (Worksheet).",
        "notes": "",
        "start_year": 2013,
        "col_var": "MARS",
        "row_var": "FLPDYR",
        "row_label": ["2013",
                      "2014",
                      "2015",
                      "2016"],
        "cpi_inflated": true,
        "col_label": ["single", "joint", "separate", "head of household", "widow", "separate"],
        "value": [[115400, 153900, 76950, 115400, 153900, 76950],
                  [117300, 156500, 78250, 117300, 156500, 78250],
                  [119200, 158900, 79450, 119200, 158900, 79450],
                  [119700, 159700, 79850, 119700, 159700, 79850]]
    },

    "_AMT_prt": {
        "long_name": "AMT exemption phaseout rate",
        "description": "AMT exemption will decrease at this rate for each dollar of AMT taxable income exceeding AMT phaseout start. ",
        "irs_ref": "Form 6251, line 29, instructions. ",
        "notes": "",
        "start_year": 2013,
        "col_var": "",
        "row_var": "FLPDYR",
        "row_label": ["2013"],
        "cpi_inflated": false,
        "col_label": "",
        "value": [0.25]
    },

    "_AMT_em": {
        "long_name": "AMT exemption amount",
        "description": "The amount of AMT taxable income exempted from AMT.",
        "irs_ref": "Form 1040, line 45, instruction (Worksheet).",
        "notes": "",
        "start_year": 2013,
        "col_var": "MARS",
        "row_var": "FLPDYR",
        "row_label": ["2013",
                      "2014",
                      "2015",
                      "2016"],
        "cpi_inflated": true,
        "col_label": ["single", "joint", "separate", "head of household", "widow", "separate"],
        "value": [[51900, 80800, 40400, 51900, 80800, 40400],
                  [52800, 82100, 41050, 52800, 82100, 41050],
                  [53600, 83400, 41700, 53600, 83400, 41700],
                  [53900, 83800, 41900, 53900, 83800, 41900]]
    },

**Comment by feenberg** [comment]
On 12/15/16, Matt Jensen <notifications@github.com> wrote:
> Is the _AMT_thd_MarriedS up to date?
>
> I don't understand the parameter well enough to tell whether it is up to
> date by examining the forms.
>
>     "_AMT_thd_MarriedS": {
>         "long_name": "Extra alminc for married sep",
>         "description": "",
>         "irs_ref": "Form 6251, line 28, instruction.",
>         "notes": "",
>         "start_year": 2013,
>         "col_var": "",
>         "row_var": "FLPDYR",
>         "row_label": ["2013",
>                       "2014"],
>         "cpi_inflated": true,
>         "col_label": "",
>         "value": [40400,
>                   41050]
>     },
>
> ```
> c62100 += cmbtp  # add income not in AGI but considered income for AMT
>     if MARS == 3 or MARS == 6:
>         amtsepadd = max(0.,
>                         min(AMT_thd_MarriedS, 0.25 * (c62100 - AMT_em_pe)))
>     else:
>         amtsepadd = 0.
>     c62100 = c62100 + amtsepadd  # AMT taxable income, which is line28
> ```
>
> Assigning this to myself but welcome input if you know the answer off of the
> top of your head.
>
> --

I have the same code in taxcalc.sas, but for 2015 the values are 41700
and 242450.

dan

**Comment by martinholmer** [comment]
The redundant policy parameter `_AMT_thd_MarriedS` was removed from `current_law_policy.json` and from the Python code by merged pull request #1270.

@MattHJensen
